modules/ROOT/pages/redelivery-policy.adoc

```diff
@@ -0,0 +1,63 @@
+= Redelivery Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, flow, redelivery, policy
+
+A redelivery policy is a filter added to Mule sources, such as modules and connectors, to prevent the dispatch of messages known to cause errors.
+
+When a redelivery policy is added to a flow's source, Mule evaluates the received data before it executes the flow's components. If the data caused failures a specified number of times in the past, the redelivery policy prevents the flow from processing the received data and raises a `REDELIVERY_EXHAUSTED` error. This behavior prevents the runtime from wasting resources by limiting the number of times Mule will execute flow messages that cause errors.
+
+Some connectors have their own logic for handling an error thrown by a redelivery policy. In such cases, check the documentation for the connector to which the source belongs. For example, the `http-connector` might return `429: TOO MANY REQUESTS`.
+
```

Comment:
"For example, the 'http-connector' might...."
> "For example, 'http-connector' might...."

Avoid using an article before a name unless the name is followed by a common noun. GLOBAL